src/format/svg/SVGData.hx

```diff
@@ -710,6 +710,16 @@ class SVGData extends Group {
 		return text;
 		
 	}
-	
+
+	private static inline function parseHex(hex:String):Int
+	{
+		if (hex.length==3) {
+			hex = hex.substr(0,1)+hex.substr(0,1)+

```

Comment:
This is on-obvious code. Please add a block comment explaining what and why (with three character hex codes, just turn `#abc# into `#aabbcc`)